pygeodiff/tests/test_single_commit.py

```diff
@@ -45,7 +45,7 @@ def basetest(
   os.remove(changeset2)
   geodiff.invert_changeset(changeset, changeset_inv)
   geodiff.apply_changeset(patched, changeset_inv)
-  geodiff.create_changeset( patched, base, changeset2 )
+  geodiff.create_changeset_dr( "sqlite", "", patched, "sqlite", "", base, changeset2 )
```

Comment:
why this is changed?

Reply:
Just to test if the new API is working from python

Reply:
maybe keep both?

Reply:
Well, the create_changeset is called several times before and this should do the same thing. But I can add a separate test for new API